src/marqo/s2_inference/model_downloading/from_s3.py

```diff
@@ -43,11 +43,14 @@ def get_s3_model_absolute_cache_path(location: S3Location) -> str:
         Returns:
             The absolute path of an s3 model if it were downloaded.
     """
-    cache_dir = os.path.expanduser(ModelCache.clip_cache_path)
+    if download_dir is None:
+        cache_dir = os.path.expanduser(ModelCache.clip_cache_path)
```

Comment:
`cache_dir = os.path.expanduser(download_dir if not None else ModelCache.clip_cache_path)`

Reply:
updated